.github/workflows/rust.yml

```diff
@@ -22,6 +22,19 @@ jobs:
     - name: Checkout Repository
       uses: actions/checkout@v3
 
+    - name: Cache Cargo Dependencies
+      uses: actions/cache@v2
```

Comment:
[GitHub Actions: All Actions will run on Node16 instead of Node12](https://github.blog/changelog/2023-05-04-github-actions-all-actions-will-run-on-node16-instead-of-node12/) で5/18に actions/cache@v2 落ちちゃうじゃんてなったので，v3にする